packages/ckeditor5-undo/docs/features/undo-redo.md

```diff
@@ -47,13 +47,13 @@ import Undo from '@ckeditor/ckeditor5-undo/src/undo';
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		// Load the plugin.
-		plugins: [ Undo, ... ],
+		plugins: [ Undo, /* ... */ ],
 
 		// Display the "Undo" and "Redo" buttons in the toolbar.
-		toolbar: [ 'undo', `redo`, ... ],
+		toolbar: [ 'undo', `redo`, /* ... */ ],
```

Comment:
`redo` is wrapped in backticks. Can we change it to single quotes to keep consistency?

Reply:
Done. :)